samples/dashboard/demos/climate/demo.css

```diff
@@ -118,19 +209,122 @@
  *
  * */
 
-.hd-cell.hd-cell-state-hover:hover > .hd-component,
-.hd-cell.hd-cell-state-active > .hd-component {
-    background-color: #5352be;
-    color: #fff;
+.hd-component-kpi,
+.hd-component-kpi-value-wrap {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+.hd-component-kpi > *,
+.hd-component-kpi-value-wrap > * {
+    align-self: center;
+    flex: none;
+    margin: auto;
+}
+
+.hd-component-kpi {
+    text-align: center;
+}
+
+.hd-component-kpi-content {
+    background: none;
+}
+
+.hd-component-kpi-value-wrap {
+    height: 100%;
+}
+
+#kpi-data .hd-component-title::before {
+    content: "📍 ";
+}
+
+#kpi-data .hd-component-kpi-value::before {
+    content: "🏔 ";
+}
+
+#kpi-data .hd-component,
+#kpi-fog .hd-component,
+#kpi-ice .hd-component,
+#kpi-max-temperature .hd-component,
+#kpi-rain .hd-component,
+#kpi-temperature .hd-component {
+    background-color: var(--color-background);
+    color: var(--color-text);
 }
 
 .hd-cell.hd-cell-state-hover {
     cursor: pointer;
 }
 
+#kpi-fog.hd-cell-state-hover:hover .hd-component,
+#kpi-fog.hd-cell-state-active .hd-component,
+#kpi-ice.hd-cell-state-hover:hover .hd-component,
+#kpi-ice.hd-cell-state-active .hd-component,
+#kpi-max-temperature.hd-cell-state-hover:hover .hd-component,
+#kpi-max-temperature.hd-cell-state-active .hd-component,
+#kpi-rain.hd-cell-state-hover:hover .hd-component,
+#kpi-rain.hd-cell-state-active .hd-component,
+#kpi-temperature.hd-cell-state-hover:hover .hd-component,
+#kpi-temperature.hd-cell-state-active .hd-component {
+    background-color: var(--color-hover);
+    color: var(--color-hover-text);
+}
+
 /* *
  *
- *  Navigator
+ *  Selection Grid
+ *
+ * */
+
+#selection-grid .hd-component {
+    background-color: var(--color-background);
+    padding: 14px;
+}
+
+#selection-grid .hc-dg-container {
+    height: 372px;
+    overflow: auto;
+}
+
+#selection-grid .hc-dg-column-header {
+    background-color: var(--color-header-background);
+    border-color: var(--color-header-border);
+    color: var(--color-header-text);
+}
+
+#selection-grid .hc-dg-outer-container {
+    background-color: var(--color-background);
+    color: var(--color-row-text);
+    position: relative;
+    top: 0 !important;
```

Comment:
The `!important` should be removed, you can change the structure of selector.

Reply:
It has to change an element style, so this is needed. In general I do not see a problem with `!important` flags in the demo as these are end-user CSS and not part of any distribution.

Reply:
The demo is a kind of showcase that would be also visible to developers. 

I can not agree that we should ignore best practices here.

Reply:
The demo is a kind of showcase that would be also visible to developers.

I can not agree that we should ignore best practices here.

Reply:
The demo is a kind of our show case and developers will have access to it too. 

Sorry, I cannot agree that we should use `!important` and break the basic rules of using CSS.

Reply:
The demo is a kind of our show case and developers will have access to it too. 

Sorry, I cannot agree that we should use `!important` and break the basic rules of using CSS.